### PR TITLE
A module for the management of ZNodes in ZooKeeper

### PR DESCRIPTION
**Pull request by treyperry**

### CODE REVIEW

File: coordination/znode [hunk 1]
```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
```
Comment (bcoca) [inline coordination/znode]: missing license

File: coordination/znode [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+DOCUMENTATION = """
+---
+module: znode
+short_description: Create, delete, retrieve, and update znodes using ZooKeeper.
+options:
+    hosts:
+        description:
+            - A list of ZooKeeper servers (format '[server]:[port]').
+        default: localhost:2181
+        required: true
```
Comment (bcoca) [inline coordination/znode:27]: default makes no sense when required: true

**Comment by jbowes** [comment]
You might like https://github.com/ansible/ansible-modules-extras/pull/310 :)

**Comment by treyperry** [comment]
@jbowes Touche. :)

**Comment by tima** [comment]
Good work @treyperry. I think the ACLs support @jbowes had in his module is important. It would be great if one of you could do that. I'll create a ticket for it if it helps.

**Comment by treyperry** [comment]
@tima Agreed. I'd be happy to help out with that. A ticket might help me keep track. :)

**Pull request merged**